pkg/taskutil/taskutil.go

```diff
@@ -42,9 +42,12 @@ func NewTask(ctx context.Context, client *containerd.Client, container container
 	if flagA {
 		logrus.Debug("attaching output instead of using the log-uri")
 		if flagT {
-			ioCreator = cio.NewCreator(cio.WithStreams(os.Stdin, os.Stdout, os.Stderr), cio.WithTerminal)
+			if con == nil {
+				return nil, errors.New("got nil con when start a container with tty")
```

Comment:
`con`  [cannot be nil](https://github.com/containerd/console/blob/v1.0.3/console.go#L77 ), one of the standard std streams (os.Stdin, os.Stdout or  os.Stderr) should always be a console otherwise program will panic

Reply:
cc @AkihiroSuda

Reply:
There seems to be no need to use `con` for input/output here，using stdio directly seems to be similar with docker.
`con` seems to be only used to resize the  terminal window

Reply:
you are using `con` here to implement the container stdio : `cio.WithStreams(con, con, nil)`, am I missing something ? `con` is a physical implementation of the terminal =>  So `con` is a terminal with a standard stdio. That means using `con` here you are setting `tty` to the standard stdio (stdout , stderr, stdin) which is good regarding the `-a` flag (you algo is good for me). My problem is with  `con == nil`; looking at [console.Current() code](https://github.com/containerd/console/blob/v1.0.3/console.go#L65-L77) the `con` can never be nil so this condition is useless.

Reply:
> There seems no need to use `con` for input/output here，using stdio seems to be similar with docker. `con` only used to resize the terminal window

@fahedouch 
sorry i didn't make it clear，What I am talking about here is this pr https://github.com/containerd/nerdctl/pull/1777